README.md

```diff
@@ -12,7 +12,12 @@ Support for translations is a
 
 They've been used successfully to translate [Comprehensive Rust 🦀].
 
-[Comprehensive Rust 🦀]: https://google.github.io/comprehensive-rust/running-the-course/translations.html
+## Use Cases
+
+Please add your project below if it uses this infrastructure for translations:
+
+- [Comprehensive Rust 🦀](https://google.github.io/comprehensive-rust/)
+- [GB ASM Tutorial](https://gbdev.io/gb-asm-tutorial/)
```

Comment:
```suggestion
- [Game Boy ASM Tutorial](https://gbdev.io/gb-asm-tutorial/)
```

Reply:
or: Game Boy Assembly Tutorial

Reply:
To be honest, I kept thinking this was something related to "Great Britain" :smile:  So I like your suggestion a lot:

```suggestion
- [Game Boy Assembly Tutorial](https://gbdev.io/gb-asm-tutorial/)
```